### PR TITLE
B 21015 int

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Swagger UI
2. run the v1 prime listmoves api - grab a move uuid
3. run the v3 prime getMoveTaskOrder api
4. verify that county returns for address objects

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

## Screenshots
Before
 
image

After

 
image

### CODE REVIEW

File: pkg/services/mto_service_item/mto_service_item_validators.go [hunk 1]
```diff
@@ -137,6 +137,12 @@ func (v *primeUpdateMTOServiceItemValidator) validate(appCtx appcontext.AppConte
 		return err
 	}
 
+	// Checks that only SIT Entry Date occurs AFTER the FADD
+	err = serviceItemData.checkSITEntryDateAndFADD(appCtx)
+	if err != nil {
+		return err
+	}
+
```
Comment (danieljordan-caci) [inline pkg/services/mto_service_item/mto_service_item_validators.go:145]: the heck - why was this taken out in `main`?

File: pkg/handlers/primeapiv3/payloads/model_to_payload.go [hunk 2]
```diff
@@ -200,6 +200,7 @@ func Address(address *models.Address) *primev3messages.Address {
 		PostalCode:     &address.PostalCode,
 		Country:        address.Country,
 		ETag:           etag.GenerateEtag(address.UpdatedAt),
+		County:         &address.County,
 	}
 }
```
Comment (danieljordan-caci) [inline pkg/handlers/primeapiv3/payloads/model_to_payload.go:205]: Can you update a test please? `TestGetMoveTaskOrder` will work!

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

File: pkg/handlers/primeapiv3/payloads/model_to_payload.go (hunk 2 above)
Comment (deandreJones) [reply pkg/handlers/primeapiv3/payloads/model_to_payload.go:205]: added

File: pkg/services/mto_service_item/mto_service_item_validators.go (hunk 1 above)
Comment (traskowskycaci) [reply pkg/services/mto_service_item/mto_service_item_validators.go:145]: I'm still seeing this in main, I remember it being a confusing thing to merge:
 https://github.com/transcom/mymove/blob/3a2885a628217ba1b6e1dcb32acf1341cb370693/pkg/services/mto_service_item/mto_service_item_validators.go#L134

**Review by traskowskycaci — APPROVED** [review]
Looks good!
image

File: pkg/handlers/primeapiv3/payloads/model_to_payload.go (hunk 2 above)
Comment (deandreJones) [reply pkg/handlers/primeapiv3/payloads/model_to_payload.go:205]: @danieljordan-caci ping

**Review by danieljordan-caci — APPROVED** [review]

**Pull request merged**